### PR TITLE
Fixes #1645 and #1644

### LINKED ISSUE #1645: Offline editor download page: "Adobe Air" should be "Adobe AIR"
The download page capitalizes "Adobe AIR" inconsistently: in some places it's "Adobe Air" and in others it's correct. They should all say "Adobe AIR"

I also think we should update the "Note for Mac Users" as follows:
- **Old:** the latest version of Scratch 2.0 Offline requires Adobe Air 20. To upgrade to Adobe Air 20 manually, go [here](https://get.adobe.com/air/).
- **Suggested:** the latest version of Scratch 2.0 Offline requires Adobe AIR 20 or newer. To upgrade Adobe AIR manually, go [here](https://get.adobe.com/air/).

### PR DESCRIPTION
**Pull request by GreenBayRules**
I changed all Adobe Air, to Adobe AIR

### CODE REVIEW

**Comment by Kenny2github** [comment]
more translator work, yay.
and now I'm helping :P
On a serious note, looks good to me!

**Comment by GreenBayRules** [comment]
Okay, added the solution too #1644 as well :D

**Comment by GreenBayRules** [comment]
Let me try an fix this...

**Comment by TheGrits** [comment]
@GreenBayRules Tip: It's generally considered good practice to make separate pull requests for separate issues, for bug tracking/code history purposes, and to allow the separate deployment of fixes.

**Comment by chrisgarrity** [comment]
@GreenBayRules @TheGrits yes it's generally better to separate PRs for different issues. In this case they're both wording changes on the same page so it fine to combine them.

File: src/views/download/l10n.json [hunk 1]
```diff
@@ -24,9 +24,10 @@
...
   "download.knownIssuesTitle": "Known issues",
-  "download.knownIssuesOne": "If your offline editor is crashing directly after Scratch is opened, install the Scratch 2 offline editor again (see step 2 above). This issue is due to a bug introduced in Adobe Air version 14 (released April 2014).",
+  "download.knownIssuesOne": "If your offline editor is crashing directly after Scratch is opened, install the Scratch 2 offline editor again (see step 2 above). This issue is due to a bug introduced in Adobe AIR version 14 (released April 2014).",
   "download.knownIssuesTwo": "Graphic effects blocks (in \"Looks\") may slow down projects due to a known Flash bug.",
   "download.knownIssuesThree": "The <b>backpack</b> is not yet available.",
+  "download.knownIssuesFour": "On Mac OS you may see a prompt indicating that \"Scratch 2 is trying to install a new helper tool\" and asking for your user name and password. We are currently investigating a solution to this problem.",
```
Comment (chrisgarrity) [inline src/views/download/l10n.json]: This also needs to get added to the 'Known Issues' section of the page: https://github.com/LLK/scratch-www/blob/18f82fce3ffecd366168740ed827392eb88a66cc/src/views/download/download.jsx#L233

The new text doesn't include any HTML tags, so you can use `FormattedMessage` for the translation.

Comment (GreenBayRules) [reply src/views/download/l10n.json]: Oh okay, ill do that :D

**Comment by GreenBayRules** [comment]
Okay all checks have passed. Can I get another review @chrisgarrity

File: src/views/download/download.jsx [hunk 2]
```diff
@@ -231,6 +231,7 @@ var Download = injectIntl(React.createClass({
                             <p><FormattedMessage id='download.knownIssuesOne' /></p>
                             <p><FormattedMessage id='download.knownIssuesTwo' /></p>
                             <p><FormattedHTMLMessage id='download.knownIssuesThree' /></p>
+                            <p><FormattedHTMLMessage id='download.knownIssuesFour' /></p>
```
Comment (chrisgarrity) [inline src/views/download/download.jsx]: Use `FormattedMessage` (no HTML) and this'll be ready.

**Comment by GreenBayRules** [comment]
Should be ready now @chrisgarrity

**Comment by GreenBayRules** [comment]
Sorry accidently closed this

**Comment by chrisgarrity** [comment]
@GreenBayRules looks good - just note that it won't get merged until after we deploy the current release that's being tested.

**Comment by GreenBayRules** [comment]
Okay :D

**Pull request merged**